drivers/dma/dma_stm32.c

```diff
@@ -36,6 +99,24 @@ static void dma_stm32_clear_stream_irq(struct device *dev, u32_t id)
 	func_ll_clear_tc[id](dma);
 	func_ll_clear_ht[id](dma);
 	stm32_dma_clear_stream_irq(dma, id);
+
+#if defined(CONFIG_DMA_STM32_V2) && defined(CONFIG_DMAMUX_STM32)
+	/* also clear the corresponding dmamux irq */
+	struct device *dev_mux =
+		 device_get_binding((const char *)"DMAMUX_1");
```

Comment:
Miss a comment to explain why this is valid on DMA_1 only

Reply:
no, with DMAMUX only
clearing the dma channel (stream) irq will also clear the corresponding dmamux channel irq